frontend/schema.graphql

```diff
@@ -0,0 +1,385 @@
+schema {
```

Comment:
Tämä tiedosto on autogeneroitua koodia -> ei kannata katselmoida sen enempää.

Reply:
Jos tämä on generoitua niin kannattaisko tää jättää suosiolla commitoimatta ja generoida tää automaattisesti tarvittaessa?

Reply:
Erittäin relevantti kysymys. 👍 Päivitin toolingin s.e. tämä generoidaan nyt automaattisesti. Ja samalla tuli mieleen että nykytoteutuksella frontin buildi feilaa jos hasura ei ole pystyssä -> täytyy vielä muuttaa scriptiä niin, että tuossa tapauksessa tarkistetaan että onko schemaa olemassa ja jos on niin jatketaan eteenpäin.